src/pages/home/report/EmojiPickerMenu/index.js

```diff
@@ -86,6 +88,18 @@ class EmojiPickerMenu extends Component {
         this.setupEventHandlers();
     }
 
+
+    componentDidUpdate(prevProps) {
+        if (
+            this.props.windowWidth !== prevProps.windowWidth
+        ) {
+            // eslint-disable-next-line react/no-did-update-set-state
```

Comment:
Is there a reason you would prefer to disable this rule. IMO it feels just as easy to create a method like `setDynamicEmojiSize` and call that method here that would do,
```
this.setState({emojiSize: dynamicEmojiSize(this.props.windowWidth)});
```
Then you wouldn't need to disable the eslint rule.